wordpress-mobile/gutenberg-mobile
# Adds isSelected to RichText component and Paragraph, Heading blocks

Fixes part of https://github.com/wordpress-mobile/gutenberg-mobile/issues/113 (for RichText only)

This PR only updates the `gutenberg` ([PR here](https://github.com/WordPress/gutenberg/pull/11857)) and `react-native-aztec` ([Aztec wrapper PR](https://github.com/wordpress-mobile/react-native-aztec/pull/73)) subprojects to make use of the corresponding additions of `isSelected` prop that propagates all the way down from Paragraph and Heading blocks down to RichText down to the Aztec RN wrapper to grab focus programmatically when one of these blocks get inserted.

![focus_insert2](https://user-images.githubusercontent.com/6597771/48483724-01ea4680-e7f3-11e8-9664-d0b4ab58c452.gif)

To test:
1. start the app
2. grab focus on any block, see the toolbar appears on the focused block
3. tap on `+` to show the block-type picker
4. tap on Heading or Paragraph blocks
5. observe the new block is inserted, the selected block is the new block and the blinking cursor is on the new block as well.
6. Repeat step 3 to 5 choosing the other block (Heading if you chose Paragraph previously, or viceversa).

**NOTE:** the focus is handled only in Paragraph and Heading blocks in this PR (both RichText-based, which in turn is powered by Aztec). Handling for other non-RichText-based blocks will be done in a separate PR.
